login.html

```diff
@@ -0,0 +1,132 @@
+    <title>SoMe One - Login</title>
+    <link rel="stylesheet" href="/dist/css/styles.css" />
+  </head>
+  <body>
+    <header>
+      <nav
+        class="navbar navbar-expand-lg navbar-dark bg-darkorange"
+        aria-label="Eighth navbar example"
+      >
+        <div class="container">
+          <a class="navbar-brand" href="index.html">SoMe One</a>
+          <button
+            class="navbar-toggler bg-darkgrey"
+            type="button"
+            data-bs-toggle="collapse"
+            data-bs-target="#navbarsExample07"
+            aria-controls="navbarsExample07"
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+          >
+            <span class="navbar-toggler-icon"></span>
+          </button>
+
+          <div class="collapse navbar-collapse" id="navbarsExample07">
+            <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+              <li class="nav-item">
+                <a class="nav-link" aria-current="page" href="index.html"
+                  >Home</a
+                >
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="profile.html">Profile</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link active" href="login.html">Login</a>
+              </li>
+            </ul>
+            <form role="search">
+              <input
+                class="form-control"
+                type="search"
+                placeholder="Search"
+                aria-label="Search"
+              />
+            </form>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <main class="">
+      <div class="row">
+        <div class="col-md-12 d-flex justify-content-center">
+          <div class="page-header pb-5">
+            <h1 class="title">My <span>profile page</span></h1>
+          </div>
+        </div>
+      </div>
+
+      <!-- Login form -->
+      <div class="px-2">
+        <div
+          class="w-80 border border-dark container-sm form-width bg-lightred rounded-2"
+        >
+          <div class="p-2">
+            <div class="mb-3">
+              <form>
+                <h1 class="h3 mb-3 fw-normal text-white">Please sign in</h1>
+
+                <div class="form-floating">
+                  <input
+                    type="email"
+                    class="form-control"
+                    id="floatingInput"
+                    placeholder="name@example.com"
+                  />
+                  <label for="floatingInput">Email address</label>
+                </div>
+                <div class="form-floating py-1">
+                  <input
+                    type="password"
+                    class="form-control"
+                    id="floatingPassword"
+                    placeholder="Password"
+                  />
+                  <label for="floatingPassword">Password</label>
+                </div>
```

Comment:
Your form inputs should include some client-side validation. Such as the `required` and `pattern` attributes.

Reply:
Added required attributes on the login page.